api/nodes/views.py

```diff
@@ -104,7 +104,7 @@ def get_draft(self, draft_id=None):
         if self.request.method not in drf_permissions.SAFE_METHODS:
             registered_and_deleted = draft.registered_node and draft.registered_node.is_deleted
 
-            if draft.registered_node and not registered_and_deleted:
+            if draft.registered_node and not draft.registered_node.is_deleted:
```

Comment:
Wait..if_registered_and_deleted is fine though.  You're allowed to get a draft that had a registration but then it was deleted.